src/language-handlebars/printer-glimmer.js

```diff
@@ -454,6 +519,31 @@ function clean(ast, newObj) {
   }
 }
 
+function countNewLines(string) {
+  /* istanbul ignore next */
+  string = typeof string === "string" ? string : "";
+  (string.match(/[\r\n]+/g) || [])[0];
```

Comment:
I don't think this line is doing anything, is it?

Reply:
It doesn't look like it, thanks for catching!